starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +20,70 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord (words) {
+  if (words.length === 0) {
+    return undefined;
+  }
```

Comment:
Para otras ocasiones, no es necesario evaluar con 0 (que no está mal si lo hacéis), cuando algo es igual a 0 javascript lo interpreta como false